test/Microsoft.TestPlatform.Extensions.BlameDataCollector.UnitTests/ProcessDumpUtilityTests.cs

```diff
@@ -138,27 +157,54 @@ public void StartProcessDumpWillThrowErrorIfProcdumpEnvVarNotSet()
         }
 
         /// <summary>
-        /// Start process dump will start exe corresponding to platform architecture
+        /// Start process dump will start exe according to Test host Process
         /// </summary>
         [TestMethod]
-        public void StartProcessDumpWillStartExeCorrespondingToPlatformArchitecture()
+        public void StartProcessDumpWillStartExeCorrespondingToTestHostProcessIn32BitOS()
         {
-            PlatformArchitecture[] platformArchitecture = { PlatformArchitecture.X64, PlatformArchitecture.X86 };
+            var guid = "guid";
+            var process = "process";
+            var processId = 12345;
+            var testResultsDirectory = "D:\\TestResults";
+
+            this.mockPlatformEnvironment.Setup(x => x.Architecture).Returns(PlatformArchitecture.X86);
+            this.mockProcessHelper.Setup(x => x.GetProcessName(processId))
+                                .Returns(process);
+            var processDumpUtility = new ProcessDumpUtility(this.mockProcessHelper.Object, this.mockFileHelper.Object, this.mockPlatformEnvironment.Object);
+
+            processDumpUtility.StartProcessDump(processId, guid, testResultsDirectory);
+
+            this.mockProcessHelper.Verify(x => x.LaunchProcess(Path.Combine("D:\\procdump", "procdump.exe"), It.IsAny<string>(), It.IsAny<string>(), null, null, null));
+        }
 
-            Dictionary<PlatformArchitecture, string> architectureExeMap = new Dictionary<PlatformArchitecture, string>()
+        /// <summary>
+        /// Start process dump will start exe according to Test host Process
+        /// </summary>
+        [TestMethod]
+        public void StartProcessDumpWillStartExeCorrespondingToTestHostProcessIn64BitOS()
```

Comment:
test the command which we are passing to procdump as well whether if its correct.